packages/codec/lib/contexts/types.ts

```diff
@@ -35,9 +28,10 @@ export interface DecoderContext {
   };
   compiler?: Compiler.CompilerVersion;
   compilationId?: string;
-  externalSolidity?: boolean; //please only set for Solidity contracts!
 }
 
+//NOTE: this is being kept for reference. but we don't actually use this
+//type anywhere anymore.
```

Comment:
could you note that "for reference" means "because the debugger is JS and won't import types"?

Reply:
Sure thing!